app/views/bulkrax/importers/index.html.erb

```diff
@@ -7,6 +7,10 @@
   </div>
 <% end %>
 
+<% content_for(:head) do %>
+  <meta name="turbolinks-cache-control" content="no-cache">
+<% end%>
```

Comment:
A few things: 

1. Generally speaking, I think `<meta>` tags are expected at the top of files. Can we move this to the top of this file? 
2. Does this `<meta>` tag stick around on other pages? I'm not sure if it gets cached at all, which could dramatically slow down other pages like the Dashboard > Collections or Dashboard > Works pages 
3. Assuming 2 ☝️ is false, can we also apply this change to the Exporters index page? It also uses DataTables

Reply:
Sounds good, I'll play around with this on our staging and report back.

Reply:
Not necessarily. As long as the `meta` tag doesn't persist on those pages I think it should be fine